config/config_test.go

```diff
@@ -261,3 +265,42 @@ streams:
 		s.Equal(test.expectedEksActivatedRules, c.Streams[0].RuntimeCfg.ActivatedRules.CisEks)
 	}
 }
+
+func (s *ConfigTestSuite) TestAWS() {
+	tests := []struct {
+		config   string
+		expected aws.ConfigAWS
+	}{
+		{
+			`
+type: cloudbeat/cis_k8s
```

Comment:
This `type` will compile obviously, but it's a bad example for someone who is trying to understand the `config.go` by going over its tests.

Please change it to `type: cloudbeat/cis_eks`

Reply:
Oh, my mistake

Reply:
@ofiriro3 this is not that simple actually
`type: cloudbeat/cis_eks` tries to connect to aws [here](https://github.com/elastic/cloudbeat/blob/main/resources/providers/awslib/config_provider.go). It returns with an error.
To support this test properly we need to inject that so it can be mocked. 
Do you think this refactor should be done as part of this pr?

* The test updated.

Reply:
Yes, I think it should be part of this PR.

https://github.com/elastic/cloudbeat/blob/main/resources/providers/awslib/config_provider_test.go
Everything is mocked here already, it should be super easy.